uwblueprint/childrens-aid-society
# Create new access_types model

## Notion ticket link
[Create new access_types model](https://www.notion.so/uwblueprintexecs/Create-new-access_types-model-a0581e50d37e4f58a9e41abed5c58752)



## Implementation description
* Created access_types.py file. Defined id and access_type fields
* Imported model into init file
* Ran db migration and updated migration file to seed db



## Steps to test
1. Access the database - docker exec -it cas_py_backend /bin/bash
2. Switch to cas db with`\c cas`
3. Run the command - SELECT * FROM access_types;
4. Verify following table appears
 id |     access_type      
----+----------------------
  1 | Twice Weekly
  2 | Weekly, Any Time
  3 | Weekly, After School
  4 | Virtual


## Checklist
